sig-governance.md

```diff
@@ -24,14 +24,21 @@ In order to standardize Special Interest Group efforts, create maximum transpare
 
 * Propose the new SIG publicly, including a brief mission statement, by emailing kubernetes-dev@googlegroups.com and kubernetes-users@googlegroups.com, then wait a couple of days for feedback
 * Ask a repo maintainer to create a github label, if one doesn't already exist: sig/foo
-* Request a new [kubernetes.slack.com](http://kubernetes.slack.com) channel (#sig-foo) from **@paris** or **@jorge**.  New users can join at [slack.kubernetes.io](http://slack.kubernetes.io).
+* Request a new [kubernetes.slack.com](http://kubernetes.slack.com) channel (#sig-foo) from [@parispittman](https://github.com/parispittman) or [@castrojo](https://github.com/castrojo).  New users can join at [slack.kubernetes.io](http://slack.kubernetes.io).
 * Slack activity is archived at [kubernetes.slackarchive.io](http://kubernetes.slackarchive.io).  To start archiving a new channel invite the slackarchive bot to the channel via `/invite @slackarchive`
 * Organize video meetings as needed. No need to wait for the [Weekly Community Video Conference](community/README.md) to discuss. Please report summary of SIG activities there.
- * Request a Zoom account from from **@paris** or **@jorge**. You must set up a google group (see below) for the SIG leads so that all the SIG leads have the ability to reset the password if necessary.
+ * Request a Zoom account from from [@parispittman](https://github.com/parispittman) or [@castrojo](https://github.com/castrojo). You must set up a google group (see below) for the SIG leads so that all the SIG leads have the ability to reset the password if necessary.
```

Comment:
Still doesn't explain how to request it, just from whom. Can you put your slack handles and say to use that (or email) to make it explicit how to contact you?